File: MachineProblems/Code/MP4.java
import java.util.Scanner;

public class MP4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int number = sc.nextInt();

        if (number <= 0) {
            System.out.println("INVALID INPUT");
            return;
        }

        if (number > 25) {
            System.out.println("EXCEEDS MAXIMUM SIZE");
            return;
        }

        for (int i = 1; i <= number; i++) {
            for (int j = 1; j <= number; j++) {
                System.out.printf("%5d ", i * j);
            }
            System.out.println();
        }

        sc.close();
    }
}

File: MachineProblems/Code/MP4.py

number = int(input())

if number <= 0:
    print("INVALID INPUT")
    quit()

if number > 25:
    print("EXCEEDS MAXIMUM SIZE")
    quit()

for i in range(1, number + 1):
    for j in range(1, number + 1):
        print(f"{i*j:5}", end=" ")
    print()